### PR TITLE
feat(react): update start and unit-tests steps

### PR DESCRIPTION
**Pull request by ehorodyski-ionic**
PR'ing to ensure this looks good. If so, I'll go ahead and refactor the remaining steps on the `feature/frameworkRedo` branch.

### CODE REVIEW

File: src/assets/data/markdown/ionic-react/start.md [hunk 1]
```diff
@@ -2,24 +2,59 @@
...
-- Restructure the starter for our base application
+- Use the CLI to create a starter application
+- Learn some Ionic CLI commands
 - Build and run the starter application
 
-## Start Here
+## Overview
 
-The Ionic CLI includes a command called `start` that we can use to bootstrap an application. The `start` command will use a basic starter application as its starting point. It is best to begin with the starter that most closely resembles the type of application we would like to create. In our case, we want a basic tabs based application, so we will use the `tabs` starter.
+THe Ionic command line is the main tool used to develop Ionic applications. This tool allows you to generate new applications, built the application, interact with Appflow and many other tasks. The command line also wraps and extends other command lines such as the Capacitor command line, react-scripts command line, etc.
```
Comment (kensodemann) [inline src/assets/data/markdown/ionic-react/start.md:11]: built the application => build the application

File: src/assets/data/markdown/ionic-react/start.md [hunk 2]
```diff
@@ -2,24 +2,59 @@
...
+
+If you type `ionic --help` at the command line you get a list of the available top level commands that can be run via the CLI. These commands are broken into two sections: Global Commands and Project Commands. Global Commands can be run from anywhere where as the Project Commands can only be run from an Ionic project directory. Common top level commands include `start`, `info`, `generate`, and `serve`. We will learn more about these commands as we use them.
```
Comment (kensodemann) [inline src/assets/data/markdown/ionic-react/start.md:13]: > Common top level commands include `start`, `info`, `generate`, and `serve`.

The "top level" bit was kinda confusing when placed right after "Global" and "Project" commands, so I removed that in the Angular one. It now reads like this:

Commonly used commands include `start`, `info`, `generate`, and `serve`.

I don't know if `generate` makes sense in the React context. You may want to find another commonly used command.

File: src/assets/data/markdown/ionic-react/unit-tests.md [hunk 3]
```diff
@@ -2,34 +2,55 @@
...
 In this lab, you will learn how to:
 
-- Run the existing suite of unit tests
+- Setup support for CI, code coverage, and snapshots
+- Create and use mock objects
 - Structure Unit Tests
-- Add component unit tests
 
-## Run the Tests
+## Setup CI, Coverage, and Snapshot Support
```
Comment (kensodemann) [inline src/assets/data/markdown/ionic-react/unit-tests.md:9]: I left it out on the existing React training, but I wonder if we shouldn't include a link to "@testing-library/react" now that we include it with the starter project.

It used to be that I had students install that and re-work the test to use it. But then on the latest go-around I noticed that the FW team already had done that in the starter at some point.

**Comment by ehorodyski-ionic** [comment]
Moving to new branch `feature/frameworkRedoReact`

**Pull request closed without merging**